apps/projects/serializers.py

```diff
@@ -0,0 +1,28 @@
+from django.utils.html import strip_tags
+from rest_framework import serializers
+
+from adhocracy4.projects.models import Project
+
+
+class AppProjectSerializer(serializers.ModelSerializer):
+
+    information = serializers.SerializerMethodField()
+    result = serializers.SerializerMethodField()
+    # todo: remove many=True once AppProjects are restricted to single module
+    published_modules = serializers.PrimaryKeyRelatedField(read_only=True,
```

Comment:
we could also just take the first published module for now. Hmm, or it might be easier to do that in the app and when we enable multi-module projects later, the app already works with a list.

Reply:
I think (as long as its possible to add several) I prefer returning all modules that are there, it feels strange to just take one of them .. and then we can see in the app what we do about it?